test/test_jit.py

```diff
@@ -7195,6 +7195,8 @@ def test(b):
             c = torch.tensor(1., requires_grad=False)
             return a, b, c  # noqa T484
 
+        torch._C._jit_override_can_fuse_on_cpu(False)
```

Comment:
@bertmaher This test was failing with my change because it was expecting flow. Hope this fix is okay.

Reply:
Yeah, seems OK.  I'm actually not. sure why this is here in test_jit, feels like it should belong in test_jit_fuser_te.